getting-started/templates/pilot-sizing.yaml

```diff
@@ -0,0 +1,891 @@
+    limits:
+      memory: "512Mi"
+
+  autoscaling:
+    enabled: true
+    minReplicas: 2
+    maxReplicas: 2
+    targetCPUUtilizationPercentage: 60
+    targetMemoryUtilizationPercentage: 70
+
+routinescheduletrigger:
+  resources:
+    requests:
+      memory: "256Mi"
+      cpu: 100m
+    limits:
+      memory: "512Mi"
+
+  replicaCount: 1
+
+  mongodb:
+    architecture: replicaset
+    replicaCount: 1
+    resources:
+      requests:
+        cpu: 150m
+        memory: 512Mi
+      limits:
+        memory: 1024Mi
+    persistence:
+      size: 1Gi
+
+saltmaster:
+  resources:
+    requests:
+      memory: "256Mi"
+      cpu: 100m
+    limits:
+      memory: "512Mi"
+
+  autoscaling:
+    enabled: true
+    minReplicas: 2
+    maxReplicas: 2
+    targetCPUUtilizationPercentage: 60
+    targetMemoryUtilizationPercentage: 70
+
+  mongodb:
+    architecture: replicaset
+    replicaCount: 1
+    resources:
+      requests:
+        cpu: 150m
+        memory: 512Mi
+      limits:
+        memory: 1024Mi
+    persistence:
+      size: 1Gi
+
+smtp:
+  resources:
+    requests:
+      memory: "256Mi"
+      cpu: 100m
+    limits:
+      memory: "512Mi"
+
+  autoscaling:
+    enabled: true
+    minReplicas: 2
+    maxReplicas: 2
+    targetCPUUtilizationPercentage: 60
+    targetMemoryUtilizationPercentage: 70
+
+swaggerapi:
+  resources:
+    requests:
+      memory: "256Mi"
+      cpu: 100m
+    limits:
+      memory: "512Mi"
+
+  autoscaling:
+    enabled: true
+    minReplicas: 2
+    maxReplicas: 2
+    targetCPUUtilizationPercentage: 60
+    targetMemoryUtilizationPercentage: 70
+
+tags:
+  resources:
+    requests:
+      memory: "256Mi"
+      cpu: 100m
+    limits:
+      memory: "512Mi"
+
+  autoscaling:
+    enabled: true
+    minReplicas: 2
+    maxReplicas: 2
+    targetCPUUtilizationPercentage: 60
+    targetMemoryUtilizationPercentage: 70
+
+  mongodb:
+    architecture: replicaset
+    replicaCount: 1
+    resources:
+      requests:
+        cpu: 150m
+        memory: 512Mi
+      limits:
+        memory: 1024Mi
+    persistence:
+      size: 1Gi
+
+taghistorian:
+  resources:
+    requests:
+      memory: "256Mi"
+      cpu: 100m
+    limits:
+      memory: "512Mi"
+
+  autoscaling:
+    enabled: true
+    minReplicas: 2
+    maxReplicas: 2
+    targetCPUUtilizationPercentage: 60
+    targetMemoryUtilizationPercentage: 70
+
+  mongodb:
+    architecture: replicaset
+    replicaCount: 1
+    resources:
+      requests:
+        cpu: 150m
+        memory: 512Mi
+      limits:
+        memory: 1024Mi
+    persistence:
+      size: 1Gi
+
+  dataRetention:
+    resources:
+      requests:
+        memory: "256Mi"
+        cpu: 100m
+      limits:
+        memory: "512Mi"
+
+    autoscaling:
+      enabled: true
+      minReplicas: 2
+      maxReplicas: 2
+      targetCPUUtilizationPercentage: 60
+      targetMemoryUtilizationPercentage: 70
+
+argoworkflows:
+  argo-workflows:
+    server:
+      replicas: 1
+
+feedservice:
```

Comment:
suggestion: Alphabetize services to make this file  easier to navigate?

Reply:
I pushed a new commit with the services alphabetized.